mmv1/third_party/terraform/.goreleaser.yml.tmpl

```diff
@@ -0,0 +1,83 @@
+archives:
+  - files:
+      # Include built binary and license files in archive
+      - 'LICENSE'
+    format: zip
+    name_template: '{{"{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"}}'
```

Comment:
wish there was a better way for these escapes... makes it very hard to read

Reply:
They're all over the resource.go.tmpl too 🙃

Reply:
The function [Delims](https://medium.com/@etiennerouzeaud/change-default-delimiters-templating-with-template-delims-857938a0b661) can change the default delimiter. If we change the delimiter, we need to make the change in all of templates?

Reply:
oh wow. Yeah we should think about doing that. Doesn't have to be now.

The downside would be we'd be deviating from standard go templating, and I don't have a pulse on how annoying that would be to contributors